src/test/prob/normal/normal_sufficient_test.hpp

```diff
@@ -0,0 +1,117 @@
+// Arguments: Doubles, Doubles, Ints, Doubles, Doubles

```

Comment:
We're on the fence with this decision.  Daniel's been
splitting functions out, but we've kept all of each
distribution together (density, cdfs, rngs).  

I think the various parameterizations of neg_binomial should
go in their own files.  That's the direction we're heading.

- Bob

> On Mar 21, 2015, at 8:42 AM, Marco Inacio <notifications@github.com> wrote:
> 
> In src/test/prob/normal/normal_sufficient_test.hpp:
> 
> > @@ -0,0 +1,117 @@
> > +// Arguments: Doubles, Doubles, Ints, Doubles, Doubles
> 
> I don't know: poisson_log and poisson_log_log tests are in the same folder, binomial_log and binomial_logit_log too, but neg_binomial_log and neg_binomial_2_log aren't.
> 
> —
> Reply to this email directly or view it on GitHub.
>